packages/flutter_driver/lib/src/find.dart

```diff
@@ -26,7 +26,7 @@ abstract class CommandWithTarget extends Command {
   }
 
   /// Deserializes the command from JSON generated by [serialize].
-  CommandWithTarget.deserialize(Map<String, String> json)
+  CommandWithTarget.deserialize(Map<String, dynamic> json)
```

Comment:
@Hixie is correct in that this is not any JSON. I should update the docs and names.